src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
```

Comment:
To create a getter property for `logo`, use the `get` keyword before the method name. For example: `get logo() { return 'Conduit logo'; }`. 🌟